soc/arm/silabs_exx32/efm32gg11b/soc_pinmap.h

```diff
@@ -18,17 +18,22 @@
 #include <soc.h>
 #include <em_gpio.h>
 
+#define GPIO_NODE DT_INST(0, silabs_gecko_gpio)
+#if DT_NODE_HAS_PROP(GPIO_NODE, swo_location)
```

Comment:
swo_location -> location_swo (also in the next line)

Reply:
Thanks for testing, should now be fixed.